src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -220,4 +235,22 @@ public static void PrintHelp(Network network)
         /// <summary>Filter peers that are within the same IP range to prevent sybil attacks.</summary>
         public bool IpRangeFiltering { get; internal set; }
     }
+
+    public static class ConnectionManagerSettingsExtensions
+    {
+        /// <summary>
+        /// Endpoint A can be mapped to endpoint B.
+        /// </summary>
+        public static bool CanBeMappedTo(this NodeServerEndpoint whiteBindEndpoint, List<NodeServerEndpoint> networkInterfacesTheNodeShouldListenOn, out NodeServerEndpoint localEndpoint)
```

Comment:
It would be better to pass type IPEndPoint as we don't make any use of NodeServerEndpoint

Reply:
You need to use IPAddress.Any and IPAddress.IPv6Any for IPv4 and IPv6 respectively to handle cases where address refers to all local interfaces (IPv4 example 0.0.0.0)